src/typography/src/UnorderedList.tsx

```diff
@@ -26,15 +28,6 @@ export default class UnorderedList extends PureComponent<any & BoxProps> {
     size: 400
   }
 
-  static styles = {
-    is: 'ul',
```

Comment:
Box no longer likes generic `string` values... we may want to change the types to support it.